toolsets/dataset/juliacode.py

```diff
@@ -0,0 +1,76 @@
+from textwrap import dedent
+
+CODE = {
+    "name": "Julia",
+    # TODO: Maybe generate libraries and setup imports from a single source of truth?
+    "libraries": """DataFrames,CSV,HTTP,JSON""",
```

Comment:
This string is provided to the LLM and should be human readable, so maybe should have spaces after the commas? It'll probably still work as is, but the LLM probably will parse better/more reliably with the spaces.